app/models/govuk_publishing_components/audit_components.rb

```diff
@@ -38,18 +48,16 @@ def initialize(path)
       }
     end
 
-  private
-
     def find_files(files, replace)
       files.map { |file| clean_file_name(file.gsub(replace, "")) }.sort
     end
 
     def clean_file_name(name)
-      name.tr("/_-", " ").gsub(".html.erb", "").gsub(".scss", "").gsub(".js", "").gsub("spec", "").gsub(".rb", "").strip
+      name.tr("/_-", " ").gsub(".html.erb", "").gsub(".erb", "").gsub(".scss", "").gsub(".js", "").gsub("spec", "").gsub(".rb", "").strip
```

Comment:
Just for readability, could this be:

```suggestion
      name
        .tr("/_-", " ")
        .gsub(".html.erb", "")
        .gsub(".erb", "")
        .gsub(".scss", "")
        .gsub(".js", "")
        .gsub("spec", "")
        .gsub(".rb", "")
        .strip
```

Reply:
Changed, thanks.